cpp/oneapi/dal/algo/knn/backend/cpu/infer_kernel_brute_force.cpp

```diff
@@ -42,7 +42,7 @@ template <typename Float, typename Task>
 static infer_result<Task> call_daal_kernel(const context_cpu &ctx,
                                            const detail::descriptor_base<Task> &desc,
                                            const table &data,
```

Comment:
For some reason `&` is on the right side, convention is to have it on the left

Reply:
due to clang-format

Reply:
Move `&` to the left hand side in the other functions and see magic